src/components/VideoDetailsCard/VideoDetailsCard.jsx

```diff
@@ -1,12 +1,22 @@
-import { useLocation, useNavigate } from 'react-router-dom';
 import './VideoDetailsCard.css';
+import { useState } from 'react';
```

Comment:
Separate local import from module import 
Ex:
```
//All module import 
import { useNavigate } from 'react-router-dom';

//All local imports
import { isAlreadyInHistory } from '../../utils';
```

Reply:
@VishakAmin thanks for the suggestion , will surely do that as code enhancement later